python/tester/upload_hsc_rc2.py

```diff
@@ -147,16 +147,16 @@ def prepare_one_visit(producer, group_id, butler, visit_id):
     )
 
     visits = set()
-    for dataId in refs.dataIds.expanded():
+    for data in refs.dataIds.expanded():
```

Comment:
`data` seems a bit confusing; it makes me think of a big blob of binary stuff. Why not `data_id`?